pom.xml

```diff
@@ -143,6 +147,12 @@ THE SOFTWARE.
       <artifactId>powermock-api-mockito</artifactId>
       <version>1.5</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.mockito</groupId>
+          <artifactId>mockito-all</artifactId>
```

Comment:
To be clear, adding this `exclusion` _does_ remove this dependency from the plugin.